### PR TITLE
[script.service.janitor] 4.4.1

### PR DESCRIPTION
**Pull request by Anthirian**
### Description
- [FIX] Improved unicode support
- [FIX] Correctly match excluded paths that contain user credentials
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.service.janitor/addon.xml [hunk 1]
```diff
@@ -39,15 +39,16 @@ Cet addon à la faculté de supprimer entièrement votre collection vidéo! Assu
         <disclaimer lang="nl_NL">Het is verstandig om videos pas permanent te verwijderen als je bekend bent met alle instellingen.</disclaimer>
         <disclaimer lang="fr_FR">Il est recommandé de ne pas supprimer physiquement de vidéos avant de bien maitriser les réglages.</disclaimer>
         <platform>all</platform>
+        <language>en nl fr</language>
```
Comment (enen92) [inline script.service.janitor/addon.xml:42]: the addon does not provide content, please remove the language tag

File: script.service.janitor/addon.xml [hunk 2]
```diff
@@ -39,15 +39,16 @@ Cet addon à la faculté de supprimer entièrement votre collection vidéo! Assu
...
         <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```
Comment (enen92) [inline script.service.janitor/addon.xml:43]: would be nice if you could use the spdx identifier for the license instead (GPL-3.0-only or GPL-3.0-or-later depending on your license

**Comment by Anthirian** [comment]
I have fixed all the issues with the .po files that I could, but still the CI bot errors out. Currently there is no way for me to avoid this construct in my code:

```
# Progress dialog translations. Keep the text between brackets as is! All dialogs have three lines we need to fill.
msgctxt "#32615"
msgid " "
msgstr "" # One space is required to remove line from previous dialog```

This is due to the way the progress dialogs work in Leia. Unfortunately if you supply a multi line text as multiple arguments and then refresh the contents of the dialog, you are forced to write at least _something_. An empty string will not cause a redraw of a line, which means I am forced to write a space in such a case. This single space is not parsed correctly by the CI bot unfortunately.

**Comment by anxdpanic** [comment]
@Anthirian 
The current issue is
```
msgstr "" # One space is required to remove line from previous dialog
```
Move the comment to above msgctxt
`"` is expected to be end of line for these cases

**Comment by Anthirian** [comment]
> 
> 
> @Anthirian
> The current issue is
> 
> ```
> msgstr "" # One space is required to remove line from previous dialog
> ```
> 
> Move the comment to above msgctxt
> `"` is expected to be end of line for these cases

Thanks, that helped actually. Are there any guidelines for what the bot expects with respect to this? I didn't run into these issues last time I pushed an update and nothing changed in terms of translations this time.

**Comment by enen92** [comment]
@Anthirian it is a known issue in addon-checker/polib regarding translator comments. We're tracking it here: https://github.com/xbmc/addon-check/issues/237

Unfortunately no fix yet. Thanks for adjusting though (to avoid the error)

**Pull request merged**